File: gems/test-miser/examples/csharp/tests/Classifier.Tests/ClassifierTests.cs
using TestMiserExample;
using Xunit;

namespace TestMiserExample.Tests;

public class ClassifierTests
{
    [Fact] public void PositivePrimary() => Assert.Equal("positive", Classifier.Classify(5));
    [Fact] public void PositiveDuplicate() => Assert.Equal("positive", Classifier.Classify(5));
    [Fact] public void High() => Assert.Equal("high", Classifier.Classify(11));
    [Fact] public void Nonpositive() => Assert.Equal("nonpositive", Classifier.Classify(0));
    [Fact] public void Smoke() => Assert.True(true);
}

File: gems/test-miser/examples/swift/Tests/ClassifierTests/ClassifierTests.swift
import XCTest
@testable import Classifier

final class ClassifierTests: XCTestCase {
    func testSmoke() { XCTAssertTrue(true) }
    func testPositivePrimary() { XCTAssertEqual("positive", classify(5)) }
    func testPositiveDuplicate() { XCTAssertEqual("positive", classify(5)) }
    func testHigh() {
        XCTAssertEqual("high", classify(11))
        XCTAssertTrue(isHigh(11))
    }
    func testNonpositive() {
        XCTAssertEqual("nonpositive", classify(-1))
        XCTAssertTrue(isNonpositive(-1))
    }
}
